providers/zfs-localpv-provisioner/run_litmus_test.yml

```diff
@@ -53,6 +53,9 @@ spec:
           - name: RECORDSIZE
```

Comment:
use appropriate comments here, can we use RECORD_SIZE instead of RECORDSIZE(it will increase readability).